### PR TITLE
Fix/test all requirements

### LINKED ISSUE #252: Requirement stop1_missing_services is not tested
### Commit
melodic-devel at 826fe36ab38362c43a31caf5fbe908215377a249

### Steps to reproduce
`rosdoc_lite prbt_hardware_support/`

### PR DESCRIPTION
**Pull request by martiniil**
Fixes #252.

### CODE REVIEW

**Comment by martiniil** [comment]
There are still rosdoc_lite warnings. But not related to this fix. See #263

File: prbt_hardware_support/test/integrationtests/integrationtest_stop1_service_missing.cpp [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+} // namespace prbt_hardware_support
+
+
+int main(int argc, char *argv[])
+{
+  ros::init(argc, argv, "integrationtest_stop1_service_missing");
+  ros::NodeHandle nh;
+
+  ros::AsyncSpinner spinner{1};
+  spinner.start();
```
Comment (agutenkunst) [inline prbt_hardware_support/test/integrationtests/integrationtest_stop1_service_missing.cpp:143]: Needed?

File: prbt_hardware_support/test/integrationtests/integrationtest_stop1_service_missing.test [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+<arg name="debug" default="false"/>
+
+<!-- Services -->
+<arg name="omit_service" default="recover"/>
+
+<arg name="recover_service_name" default="driver/recover"/>
+<arg name="halt_service_name" default="driver/halt"/>
+<arg name="hold_service_name" default="manipulator_joint_trajectory_controller/hold"/>
+<arg name="unhold_service_name" default="manipulator_joint_trajectory_controller/unhold"/>
```
Comment (agutenkunst) [inline prbt_hardware_support/test/integrationtests/integrationtest_stop1_service_missing.test]:
```suggestion
<arg name="unhold_service_name" value="manipulator_joint_trajectory_controller/unhold"/>
<arg name="halt_service_name" value="driver/halt"/>
<arg name="hold_service_name" value="manipulator_joint_trajectory_controller/hold"/>
<arg name="unhold_service_name" value="manipulator_joint_trajectory_controller/unhold"/>
```
maybe more clear

**Review by agutenkunst — CHANGES_REQUESTED** [review]

File: prbt_hardware_support/test/integrationtests/integrationtest_stop1_service_missing.cpp (hunk 1 above)
Comment (martiniil) [reply prbt_hardware_support/test/integrationtests/integrationtest_stop1_service_missing.cpp:143]: Yes I think. `ros::spin()` would block the test execution.

File: prbt_hardware_support/test/integrationtests/integrationtest_stop1_service_missing.test (hunk 2 above)
Comment (martiniil) [reply prbt_hardware_support/test/integrationtests/integrationtest_stop1_service_missing.test]: done

**Review by agutenkunst — APPROVED** [review]

**Pull request merged**